security/oidc-client-mutual-tls/README.md

```diff
@@ -0,0 +1,42 @@
+# PKCS12 keystore and truststore generation for server and client
+
+
+1. Create a keystore for the client
+
+`keytool -genkey -alias Client -keyalg RSA -keystore client-keystore.p12 -keysize 2048 -storetype PKCS12 -storepass password -dname "cn=localhost, ou=QuarkusQE, o=Redhat, L=Brno, st=BR, c=CZ"`
```

Comment:
I created certificate this way a minute ago and it has validity for 90 days (`notAfter=Nov  6 14:38:57 2023 GMT`), can you specify validity and regenerate certificates? It's not ideal to redo this often.